src/earn/EarnPoolInfoScreen.tsx

```diff
@@ -106,14 +109,127 @@ function TokenIcons({
+          })}
+        </Text>
+      </View>
+    </View>
+  )
+}
+
+function DepositAndEarningsCard({ earnPosition }: { earnPosition: EarnPosition }) {
+  const { t } = useTranslation()
+  const { balance } = earnPosition
+  const { earningItems, depositTokenId } = earnPosition.dataProps
+  const tokenInfo = useTokenInfo(depositTokenId)
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+  const localCurrencyExchangeRate = useSelector(usdToLocalCurrencyRateSelector)
+
+  // Total balance in local currency
+  const depositBalanceInUsd = tokenInfo?.priceUsd?.multipliedBy(balance)
+  const depositBalanceInLocalCurrency = new BigNumber(localCurrencyExchangeRate ?? 0).multipliedBy(
+    depositBalanceInUsd ?? 0
+  )
+  const earningItemsTokenIds = earningItems.map((item) => item.tokenId)
+  const earningItemsTokenInfo = useTokensInfo(earningItemsTokenIds)
+  const totalBalanceInLocalCurrency = depositBalanceInLocalCurrency.plus(
+    earningItems.reduce((acc, item) => {
+      if (!item.subtractFromDeposit) return acc
+      const tokenInfo = earningItemsTokenInfo.find((token) => token?.tokenId === item.tokenId)
+      const amountInUsd = tokenInfo?.priceUsd?.multipliedBy(item.amount)
+      const amountInLocalCurrency = new BigNumber(localCurrencyExchangeRate ?? 0).multipliedBy(
+        amountInUsd ?? 0
+      )
+      return acc.plus(amountInLocalCurrency ?? 0)
+    }, new BigNumber(0))
+  )
+
+  return (
+    <Card testID="DepositAndEarningsCard" cardStyle={styles.depositAndEarningCard}>
+      <View style={styles.depositAndEarningCardTitleContainer}>
+        <View style={styles.cardLineLabel}>
+          <Text numberOfLines={1} style={styles.cardTitleText}>
+            {t('earnFlow.poolInfoScreen.totalDepositAndEarnings')}
+          </Text>
+          <Touchable onPress={() => Logger.info('Title Icon Pressed!')} borderRadius={24}>
+            <InfoIcon size={16} color={Colors.gray3} />
+          </Touchable>
+        </View>
+        <View>
+          <Text style={styles.depositAndEarningCardTitleText}>
+            {t('earnFlow.poolInfoScreen.titleLocalAmountDisplay', {
+              localCurrencySymbol,
+              localCurrencyAmount: formatValueToDisplay(totalBalanceInLocalCurrency),
+            })}
+          </Text>
+        </View>
+      </View>
+
+      <View style={styles.depositAndEarningCardSubtitleContainer}>
+        <View style={styles.cardLineContainer}>
+          <View style={styles.cardLineLabel}>
+            <Text numberOfLines={1} style={styles.depositAndEarningsCardLabelText}>
+              {t('earnFlow.poolInfoScreen.deposit')}
+            </Text>
+          </View>
+          <View style={styles.flexShrink}>
+            <Text style={styles.depositAndEarningsCardValueText}>
+              {t('earnFlow.poolInfoScreen.lineItemAmountDisplay', {
+                localCurrencySymbol,
+                localCurrencyAmount: formatValueToDisplay(depositBalanceInLocalCurrency),
```

Comment:
For this value you should subtract out any earning items where `subtractFromDeposit` is true. What you have here looks to just be the pool balance, but if there is an earning line item for the compounded interest (which is included in pool balance), it should be subtracted out so that the value is not double counted.

Reply:
Fixed in 9d050848e370f4a61f5a3cc513c262ea78a7ecb9!